bigquery/google/cloud/bigquery/schema.py

```diff
@@ -146,6 +167,26 @@ def _key(self):
             self._fields,
         )
 
+    def to_standard_sql(self):
+        """Return the field as the standard SQL field representation object.
+
+        Returns:
+            An instance of :class:`~google.cloud.bigquery_v2.types.StandardSqlField`.
+        """
+        sql_type = types.StandardSqlDataType()
+        sql_type.type_kind = LEGACY_TO_STANDARD_TYPES.get(
+            self.field_type, types.StandardSqlDataType.TYPE_KIND_UNSPECIFIED
+        )
+
+        # NOTE: No need to also handle the "ARRAY" composed type, the latter
```

Comment:
It does exist, actually. It's just represented a little different. Look for `mode == "REPEATED"`

Reply:
Good to know, thanks!